package.json

```diff
@@ -32,6 +32,7 @@
     "next": "^12.2.2",
     "next-api-decorators": "^2.0.2",
     "next-auth": "^4.20.1",
+    "nft.storage": "^7.1.1",
```

Comment:
are you using this package?

```suggestion
```

I think both 
- package.json
- yarn.lock
should not be modified